src/components/LanguageButton/LanguageButton.tsx

```diff
@@ -0,0 +1,11 @@
+interface LanguageButtonProps {
+  onLanguageClick: () => void;
+}
+
+export const LanguageButton = ({ onLanguageClick }: LanguageButtonProps) => {
+  return (
+    <button className="flex h-8 w-8 items-center justify-center rounded-full bg-bg-gray-2" onClick={onLanguageClick}>
```

Comment:
It was _almost_ possible to use `RoundButton` here, but the icon inside the button was a bit too large ☹️